migrations/sql/V1.31__Drop_Mine_Detail_Table.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE mine_detail DROP CONSTRAINT mine_detail_mine_region_fkey;
```

Comment:
You may need a COMMIT; here between altering and dropping, otherwise it may fail validation for the drop because the alter will not have executed.

Reply:
Added a commit just to be safe.